### PR TITLE
[CDAP-9323] DataPrep Connections allow user to upload driver from market

### PR DESCRIPTION
**Pull request by elfenheart**
JIRA: https://issues.cask.co/browse/CDAP-9323

### CODE REVIEW

File: cdap-ui/app/cdap/components/DataPrepConnections/DatabaseConnection/DatabaseOptions.js [hunk 1]
```diff
@@ -104,17 +120,61 @@ export default class DatabaseOptions extends Component {
...
+
+    if (!jdbcConfig) { return null; }
+
+    return (
+      <span className="market-option">
+        <span
+          className="upload"
+          onClick={this.caskMarket.bind(this, db)}
+        >
+          Cask Market
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/components/DataPrepConnections/DatabaseConnection/DatabaseOptions.js]: nit: i18n?

**Review by ajainarayanan — COMMENTED** [review]
minor comment. otherwise code LGTM

**Review by ajainarayanan — COMMENTED** [review]
Code LGTM 👍

**Comment by bdmogal** [comment]
Let's close the Market modal after the upload succeeds. Other than that, works great. LGTM

**Comment by ajainarayanan** [comment]
Talked to @tonybach Decided we will pass the `displayCTA` as a prop to `AbstractWizard` which will be passed on to respective wizards instead of using `MarketStore`

**Comment by tonybach** [comment]
@bdmogal 
close-market-driver

**Review by ajainarayanan — COMMENTED** [review]
👍 Code LGTM

**Review by bdmogal — APPROVED** [review]

**Pull request merged**